test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -276,11 +316,13 @@ describe('33acrossBidAdapter:', function () {
     };
```

Comment:
seeing other functtions like `withProduct` that are not iterating over the bidRequests array, probably the change is not needed at this point but wondering if it makes sense to do it anyways so we avoid potential issues in the future.